data/contract_docs/logs.md

```diff
@@ -0,0 +1,97 @@
+# Logs
+
+Status: EARLY Draft/Proposal
+
+## LogLines
+
+### Properties and field requirements
+
+- **Time field** - _required_
+  - There must be at least one non nullable time field
+  - If there are multiple time fields present, following will decide the priority
+    - First matching time field with name `timestamp`
+    - or first matching time field with name `ts`
+    - or first matching time field
+- **Message field** - _required_
+  - There must be at lease one non nullable string field must present
+  - If more than one string fields found, the following will decide the priority
+    - First matching string field with name `body`
+    - or first matching string field with name `message`
+    - or first matching string field
+- **Severity field** - _optional_
+  - This is optional field
+  - Level/Severity of the log line can be represented with this field.
+  - This have to be a string field. (either nullable or non-nullable string field)
+  - Log field will be decided in the following order
+    - First matching string field with name `severity`
```

Comment:
for most field, the PR defines two possible names, like here, `severity` and `level`. is this necessary?
to me it seems only having a single name would make things simpler,  but maybe i'm missing something